jx_sqlite/snowflake.py

```diff
@@ -277,3 +279,26 @@ def items(self):
     @property
     def columns(self):
         return [c for cs in self.map.values() for c in cs]
+
+    def map_to_sql(self):
+        """
+        RETURN A MAP FROM THE RELATIVE AND ABSOLUTE NAME SPACE TO COLUMNS 
+        """
+        origin = self.nested_path[0]
+        fact_dict={}
+        origin_dict={}
+        for k, cs in self.map.items():
+            for c in cs :
+                if c.type not in STRUCT:
+                    if origin_dict.has_key(c.names[origin]):
+                        origin_dict[c.names[origin]].append(c)
+                    else:
+                        origin_dict[c.names[origin]] = [c]
```

Comment:
yes, I forgot about that.  you could try `origin_dict[literal_field(c.names[origin])] += [c]` which is starting to hurt the eyes. You also stick with `dict` and use `origin_dict.set_default(c.names[origin], []).append(c)`.